lib/fog/sakuracloud/requests/volume/create_disk.rb

```diff
@@ -30,7 +28,7 @@ def create_disk( name, plan, sourcearchive )
       end # Real
 
       class Mock
-        def create_disk( name, plan, sourcearchive )
+        def create_disk( name, plan, sourcearchive, options = {} )
```

Comment:
Thank you for valuable feedbacks.
It was wrong... It should be
```
      class Mock
        def create_disk( name, plan, options = {} )
```

I will commit later.